partiql-ast/src/main/resources/partiql_ast.ion

```diff
@@ -560,6 +561,22 @@ select::[
   },
 ]
 
+exclude::{
+  exprs: list::[exclude_expr],
+  _: [
+    exclude_expr::{
+      root: '.identifier.symbol',
+      steps: list::[step],
+    },
+    step::[
+      exclude_tuple_attr::{ symbol: '.identifier.symbol' },
+      exclude_collection_index::{ index: int },
+      exclude_tuple_wildcard::{},
+      exclude_collection_wildcard::{},
+    ]
```

Comment:
Why not use the available `'.expr.path'`? The steps are almost the same (with the exception that you require literals). You could just make it a parser or semantic check to verify that the root is a symbol and your steps are literals? LMK what you think.

Also, I do like how you use the spec's naming of `tuple_wildcard` and `collection_wildcard`.

Reply:
> Why not use the available `'.expr.path'`? The steps are almost the same (with the exception that you require literals). You could just make it a parser or semantic check to verify that the root is a symbol and your steps are literals? LMK what you think.

I had initially experimented with modeling these exclude expressions using `'.expr.path'` but this ended up making the schema inference on the plan a bit messier with the additional casts. I chose to model it as a separate node here to follow what the DML/DDL `path` (or simple path) had done as a separate node.

Reply:
I was under the notion that `expr`s return a PartiQL value. An individual exclude expr doesn't return a PartiQL value